bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -3157,8 +3157,8 @@
 <mpas_len_disp hgrid="mpasa60">  60000.0D0</mpas_len_disp>
 <mpas_len_disp hgrid="mpasa30">  30000.0D0</mpas_len_disp>
 <mpas_len_disp hgrid="mpasa15">  15000.0D0</mpas_len_disp>
-<mpas_len_disp hgrid="mpasa7.5">  7500.0D0</mpas_len_disp>
-<mpas_len_disp hgrid="mpasa3.75"> 3750.0D0</mpas_len_disp>
+<mpas_len_disp hgrid="mpasa7p5">  7500.0D0</mpas_len_disp>
+<mpas_len_disp hgrid="mpasa3p75"> 3750.0D0</mpas_len_disp>
```

Comment:
Smaller ask: @jedwards4b could you extend this change for the `mpas_dt` section a few lines above?